Tests/XMTPTests/CodecTests.swift

```diff
@@ -9,6 +9,10 @@ import XCTest
 @testable import XMTPiOS
 
 struct NumberCodec: ContentCodec {
+	func shouldPush(content: Double) throws -> Bool {
```

Comment:
Would you mind adding a test in here that shows you can get the shouldPush method correctly from the decodedMessage kind of like we should have fallback tests. 🙏

Reply:
seems that `Xmtp_MessageContents_EncodedContent` does not have the `shouldPush` method available which is where the fallback is coming from. Should I get it from there or from a different place?

Reply:
That was my bad from looking at JS it's on the messagev2 so wrote a test for that scenario and pushed it up. https://github.com/xmtp/xmtp-ios/pull/218/commits/1967934ae85789b62885e2850de6e008840eb97b

Reply:
thanks! @nplasterer